v2/mongodb-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/mongodb/templates/MongoDbCdcToBigQuery.java

```diff
@@ -46,22 +46,22 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * The {@link MongoDbToBigQueryCdc} pipeline is a streaming pipeline which reads data pushed to
+ * The {@link MongoDbCdcToBigQuery} pipeline is a streaming pipeline which reads data pushed to
  * PubSub from MongoDB Changestream and outputs the resulting records to BigQuery.
  *
  * <p>Check out <a
- * href="https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/mongodb-to-googlecloud/README_MongoDB_to_BigQuery_CDC.md">README</a>
+ * href="https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/mongodb-to-googlecloud/README_MongoDB_CDC_to_BigQuery.md">README</a>
  * for instructions on how to use or modify this template.
  */
 @Template(
-    name = "MongoDB_to_BigQuery_CDC",
+    name = "MongoDB_CDC_to_BigQuery",
```

Comment:
I don't think we should change the actual name, just the display data; changing the name will break existing users of the template since this is part of how it is invoked (e.g. from gcloud)

Reply:
Done thank you.